client-v2/src/shared/components/Thumbnail.tsx

```diff
@@ -1,9 +1,20 @@
 import { styled } from 'shared/constants/theme';
 
-export default styled('div')`
-  width: 130px;
-  height: 83px;
+const ThumbnailBase = styled('div')`
   background-size: cover;
   background-color: ${({ theme }) =>
     theme.shared.base.colors.backgroundColorFocused};
 `;
+
+const ThumbnailSmall = styled(ThumbnailBase)`
+  width: 83px;
+  height: 50px;
+  margin: 3px 5px 3px 5px;
```

Comment:
We should probably make an image component which can have different widths but preserves correct aspect rations and handles loading states consistently but that is perhaps for an another pr.

Reply:
👍 great idea.